src/map/skill.c

```diff
@@ -7347,8 +7347,12 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 
 	case AL_HOLYWATER:
 		if(sd) {
-			if (skill_produce_mix(sd, skill_id, ITEMID_HOLY_WATER, 0, 0, 0, 1, -1))
-				clif_skill_nodamage(src,bl,skill_id,skill_lv,1);
+			if (skill_produce_mix(sd, skill_id, ITEMID_HOLY_WATER, 0, 0, 0, 1, -1)) {
+				struct skill_unit* su;
+				if ((su = map_find_skill_unit_oncell(bl, bl->x, bl->y, NJ_SUITON, NULL, 0)) != NULL)
+					skill_delunit(su);
```

Comment:
fck off @Playtester. U need add ur knowledge.
first I said, Aqua Benedicta wasn't skill unit, u didn't believe. fkc off, I told the fact, the truth how it works.
then now, u don't believe again? whatever, u can test with stucked brain.

@secretdataz yes just need flag, I was mentioned this on previous PR (or in issue I made)
yep just use unit flag, flagged used if Aqua Benedicta was casted there, the 'cell display effect' is remain. The unit won't be deleted, I confirm it my self. Thanks to Openkore :D